src/routes/index.js

```diff
@@ -8,12 +8,15 @@ import {
 import HomePage from "../pages/HomePage";
 import NotFound from "../pages/NotFound";
 import TestState from "../pages/TestStatePage";
+import PrivateRoute from "./PrivateRoute";
 const Routing = (props) => {
 	return (
 		<Router>
 			<Routes>
 				<Route exact path="/" element={<HomePage />} />
 				<Route exact path="/test" element={<TestState />} />
+				<Route exact path="/testPrivate" element={<PrivateRoute redirectRoute="/"><TestState /></PrivateRoute>} />
+				<Route exact path="/privGame/:id" element={<PrivateRoute redirectRoute="/"><TestState /></PrivateRoute>} />
```

Comment:
You mind attaching a comment here explaining how this works so when we build on it in the future its a little easier?